options/views.py

```diff
@@ -5,8 +5,12 @@
 
 
 class OptionListView(View):
+    """
+    Sends public options as json data
```

Comment:
Change this to something along the lines of "Return a json serialized list of all public options"